internal/controller/atlasschema_controller_test.go

```diff
@@ -429,35 +429,24 @@ func TestConfigTemplate(t *testing.T) {
 	}
 	err := data.render(&buf)
 	require.NoError(t, err)
-	expected := `variable "lint_destructive" {
-  type    = bool
-  default = false
-}
-variable "lint_review" {
-  type    = string
-  default = ""
-}
-diff {
-  concurrent_index {
-    create = true
-    drop = true
-  }
-  skip {
-    drop_schema = true
-    drop_table = true
+	expected := `env "kubernetes" {
+  url     = "mysql://root:password@localhost:3306/test"
```

Comment:
The `env` block should have no label, and the `name` attribute should set to `name = atlas.env`